documentation/modules/upgrading/proc-upgrade-cluster-operator.adoc

```diff
@@ -52,4 +52,4 @@ You now have an updated Cluster Operator, but the version of Kafka running in th
 
 .What to do next
 
-Following the Cluster Operator upgrade, you can perform a xref:assembly-upgrading-kafka-versions-str[Kafka upgrade].
+Following the Cluster Operator upgrade, you can perform a xref:assembly-upgrading-kafka-versions-str[Kafka downgrade].
```

Comment:
I would just remove what to do next from this module.

Reply:
Assuming you mean just the title. Removed.

Reply:
I actually meant this whole thing! Why would a user downgrade Kafka after they've upgraded the Cluster Operator?